getting-started-spring-boot-maven/jqassistant/maven.xml

```diff
@@ -0,0 +1,29 @@
+<jqassistant-rules
+        xmlns="http://schema.jqassistant.org/rule/v2.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://schema.jqassistant.org/rule/v2.0 https://schema.jqassistant.org/rule/jqassistant-rule-v2.0.xsd">
+
+    <!-- tag::content[] -->
+    <!--1-->
+    <group id="maven:Default">
+        <includeConcept refId="maven:*"/>
```

Comment:
Same goes here. Please change to explicitly stating the concept name

Reply:
Usually I do something like this:

    <includeConstraint refId="maven:*" />
    <includeConcept refId="maven:*Report" />

The idea behind: make sure that all newly added constraints and report concepts are included without adding unnecessary concepts.